pascals-triangle.json

```diff
@@ -0,0 +1,47 @@
+{
+    "#": [
+      "Expectations are represented here as an array of arrays.",
+      "How you represent this idiomatically in your language is up to you.",
+      "An expectation of -1 indicates some sort of failure should occur"
+    ],
+    "rows": {
+      "description": "Given a count, return a collection of that many rows of pascal's triangle",
+      "cases": [
+        {
+            "description": "single row",
+            "count": 1,
+            "expected": [[1]]
+        },
+        {
+            "description": "two rows",
+            "count": 2,
+            "expected": [[1], [1, 1]]
+        },
+        {
+            "description": "three rows",
+            "count": 3,
+            "expected": [[1], [1, 1], [1, 2, 1]]
+        },
+        {
+            "description": "negative rows",
+            "count": -1,
+            "expected": -1
+        },
+        {
+            "description": "no rows",
+            "count": null,
+            "expected": -1
+        }
+      ]
+    },
+    "last_row": {
+      "description": "Returns the last row of the pascal's triangle",
```

Comment:
If dropping compatibility with old solutions is not a problem, maybe we could take the opportunity to replace the `last_row` function by a function `row` returning the *nth* row, according the mathematical convention, as mentioned [this Wikipedia page](https://en.wikipedia.org/wiki/Pascal%27s_triangle):

> The rows of Pascal's triangle are conventionally enumerated starting with row n = 0 at the top (the 0th row).